app/Rest/RestExample.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Boilerplate\Theme\Rest;
+
+use JustCoded\WP\Framework\Objects\RestController;
+
+/**
+ * Custom json rest endpoint to illustrate like it work
+ */
+class RestExample extends RestController {
```

Comment:
Make some real example, not the one, which do nothing.

For example, let's create a Controller to manage Employees. It should accept custom fields, registered with ACF, not only standard post parameters.